docs/source/features/quantization/online_rotations.md

```diff
@@ -0,0 +1,189 @@
+# Introduction
+Recent papers QuaRot and SpinQuant papers propose a quantization method for large language models (LLMs) that use rotations to simplify quantization. It quantizes all components, including weights, activations, and KV cache, into 4 bits by removing outliers without changing the output. It accomplishes this by inserting four fundamental rotations (called R1, R2, R3 and R4 in SpinQuant), into Llama models.
+
+To support these features, the ability to add online **hadamard** rotations - explicitly rotating the input before it is passed into the quantized layer - is necessary (see figure below).
+
+![Alt text]({506D28EB-F734-48C3-A3C8-0A7CF5F31141}.PNG)
```

Comment:
relocate images to under `vllm/docs/source/assets/features/quantization`. reference https://github.com/vllm-project/vllm/blob/05e1fbfc52ca575e6539de63dbb5fab929683162/docs/source/features/disagg_prefill.md?plain=1#L48 as example.